File: app/src/main/java/d/spidchenko/flashcards/db/DatabaseHandler.java
package d.spidchenko.flashcards.db;

import java.util.ArrayList;
import java.util.List;

import d.spidchenko.flashcards.data.Word;

public interface DatabaseHandler {

    void addWord(Word word);

    void updateWord(Word word);

    ArrayList<Word> getAllWords();

    void addAllWords(List<Word> words);
}

File: app/src/main/java/d/spidchenko/flashcards/db/DatabaseHandler.kt
package d.spidchenko.flashcards.db

import d.spidchenko.flashcards.data.Word

interface DatabaseHandler {
    fun addWord(word: Word)
    fun updateWord(word: Word)
    fun getAllWords(): List<Word>
    fun addAllWords(words: List<Word>)
}